api/tests/views_tests.py

```diff
@@ -111,6 +111,7 @@ def test_patch_premium_user_subdomain_cannot_be_changed(
     premium_profile = premium_user.profile
     original_subdomain = "helloworld"
     premium_profile.subdomain = original_subdomain
+    READ_ONLY_RES = "This field is read only"
```

Comment:
suggestion (blocking): until we re-use this string value another team, let's hard-code it into the line where it's used. I think `CAPITAL_VARS` are typically reserved for module-level values.